unicon_backend/constants.py

```diff
@@ -20,3 +20,8 @@ def _get_env_var(name: str, default: str | None = None, required: bool = True):
 EXCHANGE_NAME = _get_env_var("EXCHANGE_NAME", "unicon")
 TASK_QUEUE_NAME = _get_env_var("WORK_QUEUE_NAME", "unicon.tasks")
 RESULT_QUEUE_NAME = _get_env_var("RESULT_QUEUE_NAME", "unicon.results")
+
+PERMIFY_HOST = _get_env_var("PERMIFY_HOST", "http://localhost:3476")
+SCHEMA_VERSION = _get_env_var(
```

Comment:
if it is not provided could we default to the latest schema in permify db? we can get it via the [list schema](https://docs.permify.co/api-reference/schema/list-schema) endpoint

Response schema:

```json
{
  "head": "<string>",
  "schemas": [
    {
      "version": "<string>",
      "created_at": "<string>"
    }
  ],
  "continuous_token": "<string>"
}
```

`head` contains the latest schema version

Reply:
also, should it be `required=true` instead?

Reply:
good catch! swapped to required=False as it threw error for init, but the first suggestion fixes the problem 👍